src/cloudWatchLogs/explorer/cloudWatchLogsNode.ts

```diff
@@ -62,10 +62,7 @@ export class CloudWatchLogsNode extends CloudWatchLogsBase {
         super('CloudWatch Logs', regionCode, client)
     }
 
-    protected async getLogGroups(): Promise<Map<string, CloudWatchLogs.LogGroup>> {
-        return toMap(
-            await toArrayAsync(this.cloudwatchClient.describeLogGroups()),
-            configuration => configuration.logGroupName
-        )
+    public async getLogGroups(client: DefaultCloudWatchLogsClient): Promise<Map<string, CloudWatchLogs.LogGroup>> {
```

Comment:
What's the purpose of exposing `client` as a parameter?

Reply:
I am not sure. I believe this is an outdated change leftover when I was trying to use this function to get the logGroups names in the wizard.